staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapiv3/controller.go

```diff
@@ -178,6 +178,8 @@ func (c *Controller) deleteCRD(name string) {
 			if len(crdListForGV) == 0 {
 				delete(c.specsByGVandName, gv)
 			}
+			reason := "remove"
+			regenerationCounter.With(map[string]string{"group": gv.Group, "version": gv.Version, "crd": name, "reason": reason})
```

Comment:
Why split group and name?

Reply:
It would make it easier for us to filter for a specific group/version. Since CRDs are only aggregated per group/version and not all together (in OpenAPI v3), it would be a useful filter. I think it would be harder with just the crd name because of the additional regex we need to separate out the group from the crd name.